x-pack/plugins/enterprise_search/public/applications/shared/setup_guide/cloud/instructions.tsx

```diff
@@ -9,7 +9,7 @@ import { FormattedMessage } from '@kbn/i18n/react';
 import { i18n } from '@kbn/i18n';
 import { EuiPageContent, EuiSteps, EuiText, EuiLink, EuiCallOut } from '@elastic/eui';
 
-import { CLOUD_DOCS_PREFIX, ENT_SEARCH_DOCS_PREFIX } from '../../constants';
+import { docLinksService } from '../../doc_links/doc_links';
```

Comment:
nit: just import directly from the folder index.ts

```suggestion
import { docLinksService } from '../../doc_links'; 
```